packages/renderer/src/lib/deployments/DeploymentsList.spec.ts

```diff
@@ -22,7 +22,10 @@ import '@testing-library/jest-dom/vitest';
 
 import type { V1Deployment } from '@kubernetes/client-node';
 import { render, screen, within } from '@testing-library/svelte';
+/* eslint-disable import/no-duplicates */
+import { tick } from 'svelte';
 import { get } from 'svelte/store';
+/* eslint-enable import/no-duplicates */
```

Comment:
?? vite is only imported once

Reply:
it's not for vitest, it's for the block between disable/enable for svelte